src/wx/guiinit.cpp

```diff
@@ -3842,13 +3856,6 @@ bool MainFrame::BindControls()
                     NULL, &JoyPadConfigHandler[i]);
             }
 
-            // poll the joystick
-            JoystickPoller* jpoll = new JoystickPoller();
```

Comment:
This poll was for the input only. Or do you mean that If I am going to use a timer for the accels, I should disable it during gameplay? Since the events are allowed to go for extra processing anyway.

Reply:
The timer was just for the input config dialog, since the game area `OnIdle` does not run while a modal dialog is open. I'll let you think about how you want to handle accels during game play, it could be for example an additional check in the game area joystick event receiver, if the event is not mapped to game input, check accels, something like this.

If you do this, you would also need the timer for the accel config dialog for the same reason.

Reply:
We don't need to do anything for `GameArea::`. We already have `MainFrame::FilterEvent` processing keys for shortcuts that are passed to `GameArea::`. One of the changes I did here was to allow events for further processing on `GameArea`.  I believe we could just stop the timer while a game is running and then start the timer again with no game.

Reply:
Sure, that would work.